packages/react-storage/src/components/StorageManager/types.ts

```diff
@@ -32,6 +32,12 @@ export type StorageFiles = StorageFile[];
 
 export type DefaultFile = Pick<StorageFile, 'key'>;
 
+type ProcessFileParams = Required<Pick<StorageFile, 'file' | 'key'>>;
+
+type ProcessFile = (
```

Comment:
I don't know the answer, but will this be exported since the interface below is exported?